lib/models/model.js

```diff
@@ -152,7 +153,7 @@ Model.prototype._setArrayProperty = function _setArrayProperty(key, val) {
 
 
 Model.prototype.toJSON = function toJSON() {
-  var objRepresentation = mapValues(this._properties, function mapModelPropertiesToJSON(val, key) {
+  var objRepresentation = mapValues(this._properties, bind(function mapModelPropsToJSON(val, key) {
```

Comment:
Had to shorten this function name because eslint wasn't happening about how long it was.